src/main/java/org/stuartgunter/dropwizard/cassandra/CassandraFactory.java

```diff
@@ -463,4 +494,15 @@ public Cluster build(MetricRegistry metrics, HealthCheckRegistry healthChecks) {
 
         return cluster;
     }
+
+    private void validateDnsContactPointsType() {
+        if (DNS.equals(contactPointsType) && contactPoints.length > 1) {
+            throw new ValidationException("Cannot specify more than one contact point when contactPointsType is DNS.");
+        }
+    }
+
+    public enum ContactPointsType {
+        IP,
```

Comment:
Looks good to me - as long as we make clear that this is different behaviour from previous versions.